gui/src/ts/Routes.tsx

```diff
@@ -17,7 +17,7 @@ export const Routes: React.FunctionComponent = () => (
         path={Hrefs.kg(kgId).nodeSearch()}
       ></Route>
       <Route component={KgNodePage} path="/kg/:kgId/node/:nodeId" />
-      <Route exact component={KgPathPage} path={Hrefs.kg(kgId).paths} />
+      <Route component={KgPathPage} path={Hrefs.kg(kgId).paths} />
```

Comment:
This top-level page will be going away, and the paths will be viewed as part of another page, so don't sink too much effort into the top-level page.